From 1cee6bf69e7388e75ccfac6e988a2b8353af64a7 Mon Sep 17 00:00:00 2001
Message-ID: <1cee6bf69e7388e75ccfac6e988a2b8353af64a7.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 14:49:54 +0000
Subject: [PATCH v3 37/66] gnu: go-github-com-tv42-httpunix: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-tv42-httpunix): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I1b81c93afd803dfd68fae5292cb2103855cdba6d
---
 gnu/packages/golang-web.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 25 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 4cef7b6641..caccb126bc 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -2,6 +2,7 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2020 Jack Hill <jackhill@jackhill.us>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
+;;; Copyright © 2020 Nicolas Goaziou <mail@nicolasgoaziou.fr>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
@@ -831,6 +832,30 @@ (define-public go-github-com-tdewolff-parse-v2
      "This package contains several lexers and parsers written in Go.")
     (license license:expat)))
 
+(define-public go-github-com-tv42-httpunix
+  (let ((commit "2ba4b9c3382c77e7b9ea89d00746e6111d142a22")
+        (revision "0"))
+    (package
+      (name "go-github-com-tv42-httpunix")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/tv42/httpunix")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "0xbwpip2hsfhd2kd878jn5ndl8y1i9658lggha4x3xb5m1rsds9w"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "github.com/tv42/httpunix"))
+      (home-page "https://github.com/tv42/httpunix")
+      (synopsis "Go library to talk HTTP over Unix domain sockets")
+      (description "This package is a Go library to talk HTTP over Unix domain
+sockets.")
+      (license license:expat))))
+
 (define-public go-github-com-valyala-fasthttp
   (package
     (name "go-github-com-valyala-fasthttp")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index d3c6252c88..310c2ba8f4 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4334,30 +4334,6 @@ (define-public go-github-com-tomnomnom-gron
 absolute \"path\" to it.")
     (license license:expat)))
 
-(define-public go-github-com-tv42-httpunix
-  (let ((commit "2ba4b9c3382c77e7b9ea89d00746e6111d142a22")
-        (revision "0"))
-    (package
-      (name "go-github-com-tv42-httpunix")
-      (version (git-version "0.0.0" revision commit))
-      (source
-       (origin
-         (method git-fetch)
-         (uri (git-reference
-               (url "https://github.com/tv42/httpunix")
-               (commit commit)))
-         (file-name (git-file-name name version))
-         (sha256
-          (base32 "0xbwpip2hsfhd2kd878jn5ndl8y1i9658lggha4x3xb5m1rsds9w"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/tv42/httpunix"))
-      (home-page "https://github.com/tv42/httpunix")
-      (synopsis "Go library to talk HTTP over Unix domain sockets")
-      (description "This package is a Go library to talk HTTP over Unix domain
-sockets.")
-      (license license:expat))))
-
 (define-public go-github-com-blang-semver
   (let ((commit "60ec3488bfea7cca02b021d106d9911120d25fe9")
         (revision "0"))
-- 
2.41.0

